code/game/objects/objs.dm

```diff
@@ -176,6 +176,9 @@
 /atom/movable/proc/on_unset_machine(mob/user)
 	return
 
+/turf/proc/on_unset_machine(mob/user)
+	return
```

Comment:
Not appropriately documented. Please properly document this proc as per our guidelines.

Also what even is this this proc? What does it do? Why do turfs have an on_unset_machine proc that takes a mob as an argument?